.github/workflows/test.yml

```
@@ -6,7 +6,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8]
+        python-version: [3.9]
```

```suggestion
        python-version: [3.9, 3.10]
```

Is it worth testing with 3.10, since it's just been released?